config/sync/core.entity_form_display.media.image.default.yml

```diff
@@ -4,7 +4,7 @@ status: true
 dependencies:
   config:
     - field.field.media.image.field_media_image
-    - image.style.thumbnail
+    - image.style.thumbnail_115_x_115
```

Comment:
Change this back to thumbnail and create a new thumbnail with this name

Reply:
done